subsys/bluetooth/mesh/shell.c

```diff
@@ -2607,6 +2607,22 @@ static int cmd_cdb_app_key_del(const struct shell *shell, size_t argc,
 }
 #endif
 
+static int cmd_node_reset(const struct shell *shell, size_t argc, char *argv[])
+{
+	uint8_t reset;
+	int err;
+
+	err = bt_mesh_cfg_node_reset(net.net_idx, net.dst, &reset);
+	if (err) {
+		shell_error(shell, "Unable to send Node Reset (err %d)", err);
+		return 0;
+	}
+
+	shell_print(shell, "Node reset complete");
+
+	return 0;
+}
```

Comment:
Could this be merged with the reset command instead? ie keep the current reset command, but do this inside. It should be possible to reset the local device with the config message, and having two very similar commands is confusing.